src/test/java/gomedic/model/activity/ActivityIdTest.java

```diff
@@ -12,7 +12,7 @@
 public class ActivityIdTest {
     @Test
     public void constructor_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> new ActivityId(null));
+        assertThrows(NullPointerException.class, () -> new ActivityId((Integer) null));
     }
```

Comment:
Maybe we can make another test for the other constructor as well to increase code coverage? xD

```suggestion
        assertThrows(NullPointerException.class, () -> new ActivityId((Integer) null));
        assertThrows(NullPointerException.class, () -> new ActivityId((String) null));
    }
```

Reply:
yup sure, I think i tried but it gives me sth just now but yup